src/components/ExerciseForm.jsx

```diff
@@ -0,0 +1,61 @@
+import {nanoid} from 'nanoid';
+import {useRouter} from 'next/router';
+import {useState} from 'react';
+
+import useStore from '../hooks/useStore';
+
+import PlayButton from './Icons/PlayIcon';
```

Comment:
As this is an icon I would suggest to import it also as `PlayIcon `instead of `PlayButton`.
```suggestion
import PlayIcon from './Icons/PlayIcon';
```